include/keepkey/firmware/signtx_tendermint.h

```diff
@@ -0,0 +1,20 @@
+#ifndef KEEPKEY_FIRMWARE_SIGNTXTENDERMINT_H
+#define KEEPKEY_FIRMWARE_SIGNTXTENDERMINT_H
+
+#include "messages.pb.h"
+#include "trezor/crypto/bip32.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef struct _TendermintSignTx TendermintSignTx;
+
+bool tendermint_signTxInit(const HDNode *_node, const void *_msg, const size_t msgsize, const char *denom);
+bool tendermint_signTxUpdateMsgSend(const uint64_t amount, const char *to_address, const char *chainstr, const char *denom, const char *msgTypePrefix);
+bool tendermint_signTxFinalize(uint8_t *public_key, uint8_t *signature);
```

Comment:
I haven't looked at the function implementation, but any reason not to declare this function as 

`bool tendermint_signTxFinalize(uint8_t * const public_key, uint8_t * const signature);` ?

Reply:
Yes, because signature it's not a const.